README.md

```diff
@@ -2,6 +2,8 @@
 
 Remote Commands Handler is a Python application that handles remote commands via MQTT and Modbus protocols. The application listens to commands from an MQTT broker and sends corresponding commands to a Modbus server.
 
+[![Coverage Status](https://coveralls.io/repos/github/EvergenEnergy/remote-comands-handler/badge.svg?branch=coveralls)](https://coveralls.io/github/EvergenEnergy/remote-comands-handler?branch=coveralls)
```

Comment:
this is the old repo name without the added `m` - does that matter?

I haven't come across this tool before, looks very nifty!

Reply:
wow, you have an eye so well trained!
This was associated prior to the rename. I have fixed now.